docs/Project-Configuration/Project-Commands.md

```diff
@@ -606,6 +606,8 @@ EC2 Parameters:
     set. May set if `distro` is set, which will override the value from
     the distro configuration.
 -   `subnet_id` - Subnet ID for the VPC. Must be set if `ami` is set.
+-   `tenancy` - If set, defines how the hosts are distributed across
```

Comment:
might be obvious but can we say that it will be set to default if not specified?

Reply:
Sure, I mentioned the default.